plenum/test/node_request/test_propagate/test_node_lacks_finalised_requests.py

```diff
@@ -75,19 +76,10 @@ def sum_of_sent_batches():
     # Attempt to request PROPAGATEs was made as many number of times as the
     # number of sent batches in both replicas since both replicas
     # independently request PROPAGATEs
-    assert get_count(faulty_node, faulty_node.request_propagates) - \
+    assert sum_of_request_propagates(faulty_node) - \
            old_count_request_propagates == (sum_of_sent_batches() -
                                             old_sum_of_sent_batches)
 
-    requested_propagate_counts = getAllReturnVals(
-        faulty_node, faulty_node.request_propagates)
-
-    # The last attempt to request PROPAGATEs was not successful
-    assert requested_propagate_counts[0] == 0
```

Comment:
We do not need to check this anymore, because we do not call request_propagates on another replica if we've got propagates